src/GraphQL.ApiTests/GraphQL.approved.txt

```diff
@@ -1002,13 +1019,21 @@ namespace GraphQL.Introspection
     }
     public class DefaultSchemaFilter : GraphQL.Introspection.ISchemaFilter
     {
+        protected static readonly System.Threading.Tasks.Task<bool> _allowed;
+        protected static readonly System.Threading.Tasks.Task<bool> _forbidden;
```

Comment:
Improper case for protected members.  Should be Allowed and Forbidden.  Perhaps something more descriptive would be appropriate also, but it's probably fine.

Reply:
Done